WebDriverAgentLib/Categories/XCUIElement+FBAccessibility.m

```diff
@@ -42,11 +42,11 @@ - (BOOL)fb_isAccessibilityElement
   if (nil != attributeValue) {
     NSMutableDictionary *updatedValue = [NSMutableDictionary dictionaryWithDictionary:self.additionalAttributes ?: @{}];
     [updatedValue setObject:attributeValue forKey:FB_XCAXAIsElementAttribute];
-    self.additionalAttributes = updatedValue.copy;
+    self.snapshot.additionalAttributes = updatedValue.copy;
```

Comment:
then, others also need to be `self.snapshot.additionalAttributes` in here and visible

Reply:
not really due to https://github.com/appium/WebDriverAgent/blob/eca2ed1e4fd0b5a1d31ac305603b6b06faeb8a7d/WebDriverAgentLib/Routing/FBXCElementSnapshotWrapper.m#L103 magic